### PR TITLE
feat: add option to display framework icon

### PR DESCRIPTION
**Pull request by undead-voron**

### CODE REVIEW

**Comment by atinux** [comment]
Could you share a video about the result please?

**Comment by undead-voron** [comment]
https://github.com/nuxtlabs/vue-telescope-extensions/assets/7185627/46da691d-d667-4cf4-a922-a5b026d897b1

@Atinux 

Checked following stuff:

- brand icons are disabled by default
- checked brand icons on different frameworks
- checked that brand icons will return to default for opened tabs after feature is disabled
- checked brand icons when multiple tabs are active
- checked that tab state is saved after service worker has been deactivated

**Comment by undead-voron** [comment]
@Atinux 
I think I can't edit reviewers for this pull request. Can you add yourself as reviewer or should I close this pr and create a new one?

File: src/components/Settings.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<div class="relative" v-click-outside="hideList">
+    <AppButton size="small" appearance="primary" outlined @click.native="isListVisible = !isListVisible">
+        Settings
+    </AppButton>
+    <ul v-if="isListVisible" class="absolute translate-y-full border-2 border-grey-200 rounded-md mt-1 min-w-full px-2 right-0">
+        <li class="whitespace-no-wrap py-1">
+            <button @click="toggleIconState" class="whitespace-no-wrap outline-none flex flex-row items-center focus:outline-none">
+                <CheckboxSelectedIcon v-if="useFrameworkIcon" class="mr-2 w-5 h-5"/>
+                <CheckboxIcon v-else class="mr-2 w-5 h-5"/>
+                use framework brand icon
```
Comment (atinux) [inline src/components/Settings.vue]:
```suggestion
                use framework logo
```

**Comment by atinux** [comment]
Nice work @undead-voron and sorry for late answer I was on vacation.

I think we can display the icons by default and remove the ability to choose, this way we don't need to add the storage permission. Happy to update your PR?

**Comment by undead-voron** [comment]
Sure @Atinux . Please, update it if you got time.

**Comment by undead-voron** [comment]
@Atinux 
I think that we need to have storage permission anyway. As far as chrome extension has service worker in manifest version 3 (contrary to background page in manifest version 2 ) we need to store tabs state in storage (otherwise there would be no info in popup after service worker terminates).

**Comment by atinux** [comment]
I see, don't have much time right now.

I think we can keep storage but removing the settings to display the icons by default, I prefer this new behavior :)

**Comment by undead-voron** [comment]
Sounds great!

**Comment by undead-voron** [comment]
Hi @Atinux 
I had some free tine to removed settings UI for choosing framework icon and make framework icon to be displayed by default. 
here are the commits 
https://github.com/nuxtlabs/vue-telescope-extensions/pull/21/commits/567f408c769cf061ca87555feeebcfe3ae2deec9
https://github.com/nuxtlabs/vue-telescope-extensions/pull/21/commits/e036bcc3364ae69c9c2404518423bc1dc9ffc015
https://github.com/nuxtlabs/vue-telescope-extensions/pull/21/commits/4cc3cdb1526994cf8aed07cb670687f9865e3198

please, review current pr one more time, when you got time for it

**Review by atinux — APPROVED** [review]

**Pull request merged**